### PR TITLE
Adds cassocks to the chapel vendor

### PR DESCRIPTION
**Pull request by Buchse**
I decided some more tame chaplain content might be nice, so I sprited up and added a plain black cassock to the chaplain's costume vendor.
cassockfront
cassockside
cassockrear
There's nothing fancy about it, it's mechanically the same as the other chaplain robes, like the Hastur and Holiday Priest ones. 
:cl:
 * rscadd: Added cassocks to the pietyvend

### CODE REVIEW

**Comment by yclatious** [comment]
Good shit, now MAKE THE FUCKING CHANGELOG ACTUALLY SAY WHAT IT IS.

File: code/modules/clothing/head/tactical.dm [hunk 1]
```diff
@@ -12,6 +12,8 @@
 	name = "tactical helmet"
 	desc = "Standard Security gear. Protects the head from impacts. Can be attached with a flashlight."
 	icon_state = "helmet_sec"
+	species_fit = list(VOX_SHAPED)
```
Comment (jknpj) [inline code/modules/clothing/head/tactical.dm]: what is going on in here

Comment (ghost) [reply code/modules/clothing/head/tactical.dm]: He did put together some new vox sprites, but it should be in its own PR.

Comment (Buchse) [reply code/modules/clothing/head/tactical.dm]: that shouldn't be a merge issue, since that change was already merged in a previous PR I did

**Pull request merged**